RocketChat/Rocket.Chat.ReactNative
# [FIX] Cannot read property 'fetch' of undefined on ThreadMessagesView

## Proposed changes


The subscription is set the initial value as `{ }` into state [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/e883795c5fa27225d993770d5c9ca9aad0a09b0f/app/views/ThreadMessagesView/index.tsx#L84). Then we are testing the [value of subscription](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/e883795c5fa27225d993770d5c9ca9aad0a09b0f/app/views/ThreadMessagesView/index.tsx#L240) and will always be false the conditional, because it's an object.

## Issue(s)	


## How to test or reproduce

- Go to Directory View
- Open a room that you didn't join yet
- Click on the icon that move you to ThreadMessagesView

- The fetch will work properly and will not crash the app, however, you can see logging the view that is occurring an error because didn't exist the subscription in watermelon db

## Screenshots

## Types of changes




## Checklist



## Further comments

Jira: [TC-229]

[TC-229]: https://rocketchat.atlassian.net/browse/TC-229?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
